lib/reek/ast/node.rb

```diff
@@ -114,8 +115,6 @@ def look_for_types(target_types, ignoring = [], &blk)
 
       private
 
-      private_attr_reader :comments
```

Comment:
Unrelated to this PR. Boyscout rule. It drivers me **cracy** when this is not where it belongs, which is, at the beginning of the class.